events/docker/Dockerfile

```diff
@@ -8,4 +8,7 @@ RUN cd /src/events && python setup.py install
 
 RUN /src/tools/docker/bootstrap-mercurial.sh
 
+# Setup jemalloc to manage Ram more efficiently on Heroku
+ENV LD_PRELOAD="libjemalloc.so.2"
+
```

Comment:
Could you try to set narenas to 2 and see if that makes a difference?
`export MALLOC_CONF="narenas:2"`

Reply:
Or even 1, since we have a single thread

Reply:
Done in a49ce55 & pushing to testing...